packages/addon-dev/src/rollup-public-entrypoints.ts

```diff
@@ -12,10 +13,12 @@ export default function publicEntrypoints(args: {
 }): Plugin {
   return {
     name: 'addon-modules',
-    buildStart() {
-      for (let name of walkSync(args.srcDir, {
-        globs: args.include,
-      })) {
+    async buildStart() {
+      let matches = walkSync(args.srcDir, {
+        globs: [...args.include],
+      });
+
+      for (let name of matches) {
```

Comment:
I think we need to make this aware of colocation. Right now, I _think_ we'll end up calling `this.emitFile` for both the `src/components/foo.js` and `src/components/foo.hbs`.

Reply:
emit files is a no go cause it puts files in the output, and isn't compat with what we want (at least in my debugging) -- I added a bunch of comments around the current implementation